### PR TITLE
Drop items after dig a chest or locked chest

### PR DESCRIPTION
**Pull request by LNJplus**
I've copied it from minetest plus. It's untested.

### CODE REVIEW

**Comment by RHRhino** [comment]
If you drop a lot of stuff, it would be useful if an auto pick-up for items would be added first.
Calinou's version of pilzadam's item_drop mod:  
https://github.com/Calinou/carbone/blob/master/mods/item_drop/init.lua

**Comment by LNJplus** [comment]
Yeah, good idea! But I'd prefer the original mod with drop and pick-up. I think you can make a pull request. :smile:

**Comment by BlockMen** [comment]
Hmm...it might be ok to add this even without item pickup (since it would be an easy way to get rid of a chest with "trash" items).

**Comment by LNJplus** [comment]
All Right. Without item pick-up.

**Comment by LNJplus** [comment]
Tested: Works fine!

File: mods/default/nodes/chests.lua [hunk 1]
```diff
@@ -31,6 +31,21 @@ end
 
 -- Helper functions
 
+local function drop_chest_stuff()
+	return function(pos, oldnode, oldmetadata, digger)
+		local meta = minetest.env:get_meta(pos)
```
Comment (BlockMen) [inline mods/default/nodes/chests.lua]: ```minetest.env``` is deprecated

**Comment by LNJplus** [comment]
Works!

**Comment by BlockMen** [comment]
fdd34cb927750036524824af22293e6621bf0eeb

**Pull request closed without merging**